packages/handlersjs-http/lib/handlers/json-store-handler.spec.ts

```diff
@@ -0,0 +1,142 @@
+import { MemoryStore } from '@digita-ai/handlersjs-core';
+import { HttpHandlerContext } from '../models/http-handler-context';
+import { HttpHandlerResponse } from '../models/http-handler-response';
+import { JsonStoreHandler } from './json-store-handler';
+
+describe('JsonStoreHandler', () => {
+
+  interface StoreInterface {
+    data: string[];
+  }
```

Comment:
Did we try whether the typings of the handler still work if the interface has other fields as well?